package_io/input_parser.star

```diff
@@ -181,5 +194,17 @@ def default_participant():
 			"el_extra_params":			[],
 			"validator_extra_params": 	[],
 			"builder_network_params": 	None,
+			"el_min_cpu":				"",
```

Comment:
these can just be the default values that you have in every file?

Reply:
Wouldn't be able to use a global default for all. As I would potentially wanna do individual limits based on client performance.

Reply:
alright, if we have the same values now I would have preferred just having it all here. for different values perhaps we could have had a map in the input_parser; 

```
el_min_resources = {
   "geth": {"cpu" : 334}
}
```

On the otherhand why are these `""` - can you make them `0` instead?

Reply:
as far as I know 0 would set the defaults to "unlimited" resources.

Reply:
geth is already using a different default than besu/nethermind and erigon.

Reply:
Okay on the have a `{}` that stores client specifics - you can have a look at https://github.com/kurtosis-tech/eth-network-package/blob/main/package_io/constants.star#L16-L22 - but I think we can tackle this separately in a separate PR; while we get this in

Where-ever downstream you are comparing to `""` and then setting the min/max you would just compare to `0` and set the right value. What I find off is that the the thing goes from a string `""` to a number. Having same types just feels nicer. Say someone looks at input parser and they enter `"55"` instead of `55`. Will that work? Can't it just be `0`?

Reply:
updated according to this. Pls review again.

Reply:
You happy with the current state? Can this be resolved?

Reply:
@barnabasbusa having a look.